### DISCUSSION TITLE
OCAPI Integration : Multi-Currency Multi-Region Single Site

### DISCUSSION THREAD

**Discussion by Nikhileshinps** [SalesforceCommerceCloud/sfcc-ci#464] [Team Posts]
Hi Team,

We are working on a project where we want to integrate with a Multi-Currency and Multi-Region Site. When we add the product to the basket we see following flash message : 

ProductItemWithoutPrice

While creating the basket we are setting the currency. 

Has anyone encounter this message before ?

On the website it uses session to change the currency and assign the pricebook.
We are not using the same session in the OCAPI integration.
 
Regards,
Nikhilesh

**Comment by pravngaur** [comment]
Hi,

Please have a look at
https://documentation.b2c.commercecloud.salesforce.com/DOC1/index.jsp


   - ProductItemWithoutPrice - indicates that the item has no price.

If you are able to add this same product from storefront but not via okapi
call, I would recommend validating your request itself - and if it does not
work from storefront also, then please configure the price for the given
product.


On Fri, Sep 6, 2019 at 3:59 AM Nikhileshinps <notifications@github.com>
wrote:

> Hi Team,
>
> We are working on a project where we want to integrate with a
> Multi-Currency and Multi-Region Site. When we add the product to the basket
> we see following flash message :
>
> ProductItemWithoutPrice
>
> While creating the basket we are setting the currency.
>
> Has anyone encounter this message before ?
>
> On the website it uses session to change the currency and assign the
> pricebook.
> We are not using the same session in the OCAPI integration.
>
> Regards,
> Nikhilesh
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/SalesforceCommerceCloud/teams/community/discussions/111?email_source=notifications&email_token=AKECNZOUFN3KWTK5FAGQZPDQIF7DPA5CNFSM4IUC57HKYY3PNVWWK3TUL52HS4DFVZCGS43DOVZXG2LPNZIG643UVJRW63LNMVXHIX3JMTHAAAJS7A>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKECNZLUZB277ZSSSHX26XTQIF7DPANCNFSM4IUC57HA>
> .
>


-- 
Thanks and Regards,
Praveen Gaur
+91-9953335721

**Comment by aventurinisalesforce** [comment]
in the session is there the good pricebook related to the selected currency?
